ubuntudesign/www.ubuntu.com
# Tiny robotics tweak

## Done

Added an eight-col to a paragraph so that vertical spacing is correct at smaller viewports

## QA

View /internet-of-things/robotics at a smaller viewport and see that the "Ubuntu runs on the robotic platforms that matter" row has the correct vertical spacing at the bottom.
